doc/introduction/measurements.rst

```diff
@@ -41,6 +41,7 @@ The available measurement functions are
     ~pennylane.purity
     ~pennylane.classical_shadow
     ~pennylane.shadow_expval
+    ~pennylane.measure
```

Comment:
```suggestion
```
Not a major point, but I interpret this list to be the objects that can be returned by a QNode. But mid-circuit measurements cannot be.

Reply:
Good point, thanks!